src/data/markdown/docs/02 javascript api/07 k6-experimental/02 redis.md

```diff
@@ -27,7 +27,4 @@ which operate in a synchronous manner,
 the Redis `Client` operates in an asynchronous manner.
 In practice, this means that using the Redis `Client`'s methods won't block test execution,
 and that the test will continue to run even if the Redis `Client` isn't ready to respond to the request.
-
-This new execution model does introduce a potential caveat: to depend on operations against Redis, all following operations should be made in the context of a [promise chain](https://javascript.info/promise-chaining).
-As the examples below demonstrate, whenever there is a dependency on a Redis operation result or return value, the operation should be wrapped in a promise itself.
-This way, a user can perform asynchronous interactions with Redis in a seemingly synchronous manner.
\ No newline at end of file
+However, achieving a seemingly synchronous behavior can be done using the `async/await` syntax. When preceding an asynchronous Redis `Client` operation with the `await` keyword in the context of an `async` function, the execution of the function will wait for the operation to complete before continuing its execution.
```

Comment:
```suggestion
The `async/await` syntax can be used to make the code look synchronose while still not blocking the test execution.
```

I still think that we either:
1. don't go into details of how to await/async works and link to MDN 
2. or we make a page where we explain it once and link to it.

I kind of feel like we won't do better job then MDN does, but maybe having our own page has some benefits s well :shrug:

Reply:
Rephrased following your proposal 👍🏻